lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,20 +1,32 @@
 <div class="section">
     <div class="course-meta-container col-12 md-col-8 sm-col-12">
-        <a href="<%- course_url %>" class="course-image-link">
-            <img
-                class="header-img"
-                src="<%- course_image_url %>"
-                alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
-        </a>
+        <div class="course-image-container">
+            <% if (course_url){ %>
+                <a href="<%- course_url %>" class="course-image-link">
+            <% } %>
+                <img
+                    class="header-img"
+                    src="<%- course_image_url %>"
+                    alt="<%= interpolate(gettext('%(courseName)s Home Page.'), {courseName: display_name}, true) %>"/>
```

Comment:
The current alt text describes where the link will take the user to. If there is no link it is misleading. This is not hover text, it is text that a screenreader user will hear.